binding/dotnet/PvSpeaker/PvSpeaker.cs

```diff
@@ -62,7 +62,7 @@ private static IntPtr ImportResolver(string libraryName, Assembly assembly, DllI
 #pragma warning disable IDE0059
 
             IntPtr libHandle = IntPtr.Zero;
-            NativeLibrary.TryLoad(GetLibraryPath(), out libHandle);
+            NativeLibrary.TryLoad(Utils.PvLibraryPath(libraryName), out libHandle);
```

Comment:
Can we execute this code to init a static var instead of doing it here? I remember we saw that this gets executed every single time a dll function is called, while really it only needs to be called once. We could do something like:
```csharp
private static string _libraryPath = Utils.PvLibraryPath(libraryName);
```